BackgroundChanger.js

```diff
@@ -0,0 +1,31 @@
+const TIME = 10;
```

Comment:
No problem! I'll change the name to `TOTAL_TIME` so that it won't be mistaken for anything.